processing file asmFiles/coherence2.asm (1 of 3)
processing file asmFiles/coherence3.asm (2 of 3)
processing file asmFiles/coherence.asm (3 of 3)
--- memsim.hex	2019-11-18 18:57:25.380682678 -0500
+++ memcpu.hex	2019-11-18 18:57:25.355681933 -0500
@@ -4,6 +4,7 @@
 :04000300AC4300000A
 :04000400FFFFFFFFFC
 :04002000000000F4E8
+:0400210000000F00CC
 :04008000340100F453
 :0400810034040080C3
 :04008500AC8100004A
processing file asmFiles/test.coherence1.asm (1 of 2)
--- memsim.hex	2019-11-18 18:57:33.362920436 -0500
+++ memcpu.hex	2019-11-18 18:57:33.316919065 -0500
@@ -17,7 +17,7 @@
 :04008700AD090000BF
 :04008800FFFFFFFF78
 :04010000DEADBEEFC3
-:0401010089ABCDEF0A
+:04010100BAD0BAD0E6
 :040102000123456729
 :04010300CAB1FEED92
 :00000001FF
processing file asmFiles/test.coherence2.asm (2 of 2)
--- memsim.hex	2019-11-18 18:57:34.540955523 -0500
+++ memcpu.hex	2019-11-18 18:57:34.509954601 -0500
@@ -19,7 +19,5 @@
 :04008600AD2A00009F
 :04008700FFFFFFFF79
 :0401000000000001FA
-:04010200DEADBEEFC1
-:040103000000DEAD6D
 :040105000000BEEF49
 :00000001FF
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
--- memsim.hex	2019-11-18 18:57:53.618523764 -0500
+++ memcpu.hex	2019-11-18 18:57:53.570522337 -0500
@@ -98,134 +98,69 @@
 :0400C200080000BC76
 :0400C30003E000084E
 :0401000000000001FA
-:0401010000000040BA
-:0401020000000001F8
-:0401030000000003F5
-:0401040000000004F3
-:0401050000000007EF
-:040106000000000BEA
-:040107000000000CE8
-:0401080000000015DE
-:0401090000000016DC
-:04010A0000000016DB
-:04010B0000000019D7
-:04010C000000001BD4
-:04010D000000001DD1
-:04010E0000000020CD
-:04010F0000000021CB
-:0401100000000029C2
-:0401110000000033B7
-:0401120000000036B3
-:0401130000000038B0
-:040114000000003DAA
-:040115000000003DA9
-:040116000000003EA7
-:0401170000000042A2
-:04011800000000479C
-:040119000000004B97
-:04011A000000004D94
-:04011B000000005090
-:04011C00000000518E
-:04011D00000000518D
-:04011E00000000538A
-:04011F000000005A82
-:040120000000006378
-:040121000000006476
-:0401220000000007D2
-:0401230000000007D1
-:0401240000000007D0
-:0401250000000008CE
-:0401260000000008CD
-:0401270000000008CC
-:0401280000000009CA
-:040129000000000AC8
-:04012A000000000FC2
-:04012B0000000010C0
-:04012C0000000011BE
-:04012D0000000017B7
-:04012E000000001AB3
-:04012F0000000020AC
-:0401300000000020AB
-:0401310000000023A7
-:0401320000000024A5
-:0401330000000026A2
-:0401340000000026A1
-:04013500000000289E
-:040136000000002F96
-:040137000000003094
-:040138000000003291
-:040139000000004181
-:04013A00000000427F
-:04013B00000000437D
-:04013C00000000447B
-:04013D000000005965
-:04013E000000005B62
-:04013F000000005C60
-:040140000000005C5F
-:040141000000006456
-:04018000000000017A
-:040181000000000377
-:040182000000000475
-:040183000000000771
-:040184000000000770
-:04018500000000076F
-:04018600000000076E
-:04018700000000086C
-:04018800000000086B
-:04018900000000086A
-:04018A000000000968
-:04018B000000000A66
-:04018C000000000B64
-:04018D000000000C62
-:04018E000000000F5E
-:04018F00000000105C
-:04019000000000115A
-:040191000000001555
-:040192000000001653
-:040193000000001652
-:040194000000001750
-:04019500000000194D
-:040196000000001A4B
-:040197000000001B49
-:040198000000001D46
-:040199000000002042
-:04019A000000002041
-:04019B000000002040
-:04019C00000000213E
-:04019D00000000233B
-:04019E000000002439
-:04019F000000002636
-:0401A0000000002635
-:0401A1000000002832
-:0401A2000000002930
-:0401A3000000002F29
-:0401A4000000003027
-:0401A5000000003224
-:0401A6000000003322
-:0401A700000000361E
-:0401A800000000381B
-:0401A9000000003D15
-:0401AA000000003D14
-:0401AB000000003E12
-:0401AC00000000410E
-:0401AD00000000420C
-:0401AE00000000420B
-:0401AF000000004309
-:0401B0000000004407
-:0401B1000000004703
-:0401B2000000004BFE
-:0401B3000000004DFB
-:0401B40000000050F7
-:0401B50000000051F5
-:0401B60000000051F4
-:0401B70000000053F1
-:0401B80000000059EA
-:0401B9000000005AE8
-:0401BA000000005BE6
-:0401BB000000005CE4
-:0401BC000000005CE3
-:0401BD0000000063DB
-:0401BE0000000064D9
-:0401BF0000000064D8
+:040102000000005A9F
+:0401030000000051A7
+:0401040000000033C4
+:0401050000000019DD
+:0401060000000050A5
+:0401070000000029CB
+:0401080000000016DD
+:0401090000000015DD
+:04010A000000000CE5
+:04010B000000003EB2
+:04010C000000004BA4
+:04010D0000000047A7
+:04010E00000000539A
+:04010F00000000519B
+:040110000000004D9E
+:0401110000000016D4
+:040112000000000BDE
+:040113000000001DCB
+:0401140000000007E0
+:0401150000000021C5
+:040116000000006382
+:040117000000001BC9
+:04011800000000647F
+:0401190000000042A0
+:04011A000000003DA4
+:04011B0000000020C0
+:04011C0000000001DE
+:04011D0000000036A8
+:04011E0000000004D9
+:04011F000000003D9F
+:0401200000000038A3
+:0401210000000003D7
+:0401220000000030A9
+:0401230000000008D0
+:040124000000004295
+:040125000000006472
+:040126000000000FC6
+:040127000000005C78
+:040128000000004192
+:0401290000000020B2
+:04012A0000000009C8
+:04012B000000002FA1
+:04012C000000005976
+:04012D0000000011BD
+:04012E0000000007C6
+:04012F0000000023A9
+:040130000000004487
+:0401310000000020AA
+:040132000000000ABF
+:0401330000000007C1
+:0401340000000017B0
+:040135000000005C6A
+:040136000000005B6A
+:04013700000000289C
+:040138000000001AA9
+:0401390000000008BA
+:04013A00000000249D
+:04013B00000000269A
+:04013C0000000008B7
+:04013D000000002698
+:04013E0000000010AD
+:04013F00000000328A
+:0401400000000007B4
+:040141000000004377
 :043FFE0000000600B9
 :00000001FF
processing file asmFiles/example.asm (1 of 1)
--- memsim.hex	2019-11-18 18:57:59.795707755 -0500
+++ memcpu.hex	2019-11-18 18:57:59.741706148 -0500
@@ -39,6 +39,7 @@
 :04008E0027BD000486
 :04008F0003E0000882
 :040090000000000369
+:04009100000000016A
 :040FFE0000000008E7
 :041FFE0000000208D5
 :00000001FF
